### PR TITLE
Mkdocs

### LINKED ISSUE #178: Make the Docs folder navigable
We need to be able to navigate around the documentation in the docs folder.  

Also:  we need a link to the docs on the Readme for this repository

The docs are being published to GitHub pages at:  https://fritzandfriends.github.io/BlazorWebFormsComponents/

### PR DESCRIPTION
**Pull request by crazy-max**
Closes #178 

@csharpfritz Generated documentation will be pushed to gh-pages branch so you will have to switch branch in GitHub Pages settings.

### CODE REVIEW

File: .github/workflows/docs.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+name: docs
+
+on:
+  push:
+    branches:
+      - 'main'
+      - 'mkdocs' # Remove after merge
```
Comment (crazy-max) [inline .github/workflows/docs.yml:7]: Remove this line when merged (was here for testing purpose and demo)

File: .github/workflows/docs.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+          sudo chown -R $(id -u):$(id -g) ./site
+      -
+        name: Check GitHub Pages status
+        uses: crazy-max/ghaction-github-status@v1
+        with:
+          pages_threshold: major_outage
+      -
+        name: Deploy
+        #if: success() && github.event_name != 'pull_request' && (endsWith(github.ref, github.event.repository.default_branch) || steps.prepare.outputs.release == 'true')
+        if: success() && github.event_name != 'pull_request' # Remove after merge and use the previous condition
```
Comment (crazy-max) [inline .github/workflows/docs.yml:63]: Remove this line when merged and uncomment the previous one (was here for testing purpose and demo)

**Comment by csharpfritz** [comment]
Thank you very much for  the contribution to make our documentation BETTER

**Comment by crazy-max** [comment]
@csharpfritz :rocket:

**Pull request merged**